src/Infrastructure.EntityFramework/Vault/Repositories/CipherRepository.cs

```diff
@@ -303,12 +303,12 @@ public async Task DeleteDeletedAsync(DateTime deletedDateBefore)
         }
     }
 
-    public async Task<CipherDetails> GetByIdAsync(Guid id, Guid userId, bool useFlexibleCollections)
+    public async Task<CipherDetails> GetByIdAsync(Guid id, Guid userId)
     {
         using (var scope = ServiceScopeFactory.CreateScope())
         {
             var dbContext = GetDatabaseContext(scope);
-            var userCipherDetails = new UserCipherDetailsQuery(userId, useFlexibleCollections);
+            var userCipherDetails = new UserCipherDetailsQuery(userId, false);
```

Comment:
💭 Just wanted to double check that you wanted to pass in the `false` value here. Initial reaction is we should be passing `true` but if we've already updated the underlying `vNext vs vCurrent` then `false` makes sense.

Reply:
It's intentional - because this flag was never used, this _never_ evaluated to `true`, and these v2 sprocs were never used.

I haven't removed this parameter completely because it's used by other callers that I haven't updated yet.

More detail is in the [tech breakdown](https://bitwarden.atlassian.net/wiki/spaces/EN/pages/900825246/Tech+Breakdown+Flexible+Collections+MVP+cleanup#Remove-AccessAll-from-the-database). These sprocs fall into the "third category" listed in that section. Let's sync up on this soon.

Reply:
Makes sense! Thanks for explaining.